src/app/effects/backend.wired.effects.ts

```diff
@@ -282,10 +283,6 @@ export class BackendWiredEffects {
                 const file: FileSystemFileHandle = await response[0];
                 let content : any = await file.getFile();
                 content = await content.text();
-                // create writable and destroy it
-                const writable = await file.createWritable();
```

Comment:
this destroys the feature that save a project

Reply:
works for me.. 'Save' creates a new writer